hydra-node/test/Hydra/Network/ReliabilitySpec.hs

```diff
@@ -126,17 +130,20 @@ spec = parallel $ do
             bobReceived <- toList <$> readTVarIO messagesReceivedByBob
             pure (aliceReceived, bobReceived, logs)
          in
-          within 1000000 $ msgReceivedByBob
-            === aliceToBobMessages
-            & counterexample (unlines $ show <$> reverse traces)
-            & tabulate "Messages from Alice to Bob" ["< " <> show ((length msgReceivedByBob `div` 10 + 1) * 10)]
-            & tabulate "Messages from Bob to Alice" ["< " <> show ((length msgReceivedByAlice `div` 10 + 1) * 10)]
+          within 1000000 $
```

Comment:
Why is this changed? Seems like some code formatter on one of our machines somewhere is not doing its job